### PR TITLE
Sub navigation

### PR DESCRIPTION
**Pull request by Ne-anna**
Created sub-navigation.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -288,13 +282,31 @@ nav {
...
 .sub-navigation__button--back img {
   width: 17px;
   height: 17px;
 }
 
+.sub-navigation__text {
+  font-family: "Axis";
+  color: var(--black);
+  text-decoration: none;
+  transition: 0.7s ease;
```
Comment (DarinaBlazevica) [inline style.css:303]: Why are you having transition here?

File: style.css [hunk 2]
```diff
@@ -288,13 +282,31 @@ nav {
...
-  margin-right: 8px;
 }
+
 .sub-navigation__button--back img {
   width: 17px;
   height: 17px;
 }
 
+.sub-navigation__text {
+  font-family: "Axis";
```
Comment (DarinaBlazevica) [inline style.css]: You can use variable for it as well as for color

File: script.js [hunk 3]
```diff
@@ -6,37 +6,38 @@ const navigationMenu = document.querySelector(".navigation__menu");
...
 
 function openCart() {
     addToCart.classList.toggle("active");
     navigationMenu.classList.remove("active");
     hamburger.classList.remove("active");
     body.classList.remove("fixed-body");
+    subMenu.classList.remove("active");
 }
 
 function fixedBody() {
```
Comment (DarinaBlazevica) [inline script.js:35]: Is this function for scroll enabling  when menu/sub-menu is closed  ? 
If yes, maybe try to use naming enableScroll? In this case people will better understand what you're doing inside specific function

File: script.js [hunk 4]
```diff
@@ -46,6 +47,39 @@ function fixedBody() {
...
+    subMenu.classList.remove("active");
+}
+
+backNavigationWord.onclick = () => {
+    navigationMenu.classList.add("active");
+    subMenu.classList.remove("active");
+}
+
 let slidePosition = 2;
 const slides = document.getElementsByClassName('carousel__item');
```
Comment (DarinaBlazevica) [inline script.js:84]: Try to use getElementById instead of getElementsByClassName

File: style.css [hunk 5]
```diff
@@ -237,6 +238,63 @@ nav {
   font-weight: bolder;
 }
 
+.sub-navigation {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
```
Comment (DarinaBlazevica) [inline style.css:250]: Why are you using this there ? Is it possible to have the same results without defining list-style-type: none. Please try

File: style.css (hunk 1 above)
Comment (Ne-anna) [reply style.css:303]: When changing to sub-menu, with transition it looks smoother and better in my opinion.

File: script.js [hunk 6]
```diff
@@ -46,6 +47,39 @@ function fixedBody() {
...
+    navigationMenu.classList.remove("active");
+}
+
+backNavigationButton.onclick = () => {
+    fixedBody();
+    navigationMenu.classList.add("active");
+    subMenu.classList.remove("active");
+}
+
+backNavigationWord.onclick = () => {
```
Comment (DarinaBlazevica) [inline script.js]: I advice you to use backBtn in this case

Comment (DarinaBlazevica) [reply script.js]: Or backButton (something like this)

**Review by ilze-dom — APPROVED** [review]

File: style.css [hunk 7]
```diff
@@ -227,16 +227,86 @@ nav {
...
+}
+
+.sub-navigation__text {
+  font-family: "Axis";
+  color: var(--black);
+  text-decoration: none;
+  transition: 0.7s ease;
+  margin: 0;
+  letter-spacing: 2px;
+  padding: 18px 0px;
```
Comment (EgijaLocmele) [inline style.css]: No need to add px for 0

**Pull request merged**